get_recipe_price.py

```diff
@@ -0,0 +1,17 @@
+#submitted by Keren Or Hadad 208025205
+
+
+def get_recipe_price(d_ing: dict,  *args, optionals = []):
```

Comment:
Add documantation, and please use more informative names for your parameters.
(Notice that there were specific names that you have been requested to use).

Reply:
done